sample-apps/async-example/README.md

```diff
@@ -1,6 +1,29 @@
 ## Setup instructions
 
-1. Install composer dependencies
+1. Install composer dependencies with `composer install`
 2. Copy `config-sample.php` and rename to `config.php`
 3. Insert your API key from Raygun into the `API_KEY` field
 4. Submit the form with empty/zero values to trigger errors
+
+### Docker
+
+This sample app is set up to run at http://localhost:8888 by default. You can change the port in the `docker-compose.yml` file.
+
+1. Ensure dependencies are installed (skip this step if this has been done already):
+
+```
+composer install
+```
+
+2. Using docker-compose:
+
+```
+docker-compose up
```

Comment:
I think we might need to add an extra step here to do `docker-compose build` first. In my experience I had an error when opening the browser because Docker couldn't find the vendor autoload script needed to load raygun4php dependencies. This was because I installed the dependencies outside of the `src` folder before setting up Docker. I added them to `src` but continued to get a not found error until I tried building.

Probably a pretty obvious step but it did get me stuck 😅 would be good to include it, as you need to run `docker-compose build` every time you make changes before running `docker-compose up` to see the changes.

Reply:
I found out you can do this in one line by running it with the build option: `docker-compose up --build`